services/brig/src/Brig/Run.hs

```diff
@@ -112,9 +112,9 @@ mkApp o = do
     middleware :: Env -> (RequestId -> Wai.Application) -> Wai.Application
     middleware e =
       Metrics.servantPlusWAIPrometheusMiddleware (sitemap o) (Proxy @ServantCombinedAPI)
-        . catchErrors (e ^. applog) [Right $ e ^. metrics]
         . GZip.gunzip
         . GZip.gzip GZip.def
+        . catchErrors (e ^. applog) [Right $ e ^. metrics]
```

Comment:
@pcapriotti There is a separate PR for this change, right?

Reply:
Yes, that change should disappear when we merge develop.